docs/keycloak.md

```diff
@@ -0,0 +1,42 @@
+# Configuring Keycloak for Gatekeeper
+
+## Prerequisites
+- You need access to a Keycloak instance and to configure a realm.
+
+## TL;DR:
+- Create a client and set its "Access Type" to "Confidential". The name of the client is the CLIENT_ID and the CLIENT_SECRET is in the credentials tab of the client.
+- Configure the client with your <gatekeeper url>/callback as "Valid Redirect URIs" and your frontend URL as "Web Origins"
+- The DISCOVERY_URL is the endpoint "OpenID Endpoint Configuration" found on the "Realm Settings" page. (Right click and copy link)
+
+### Create and configure the Client
+
+#### Create the client
+1. Log in to Keycloak and select your realm.
+2. Select "Clients" in the menu on the left and click "Create" in the top right corner of the screen.
+3. Enter a client ID. Note this down, this is the CLIENT_ID. Ensure the Client Protocol is "OpenID Connect".
+4. Click "Save"?.

```

Comment:
`?.` ?

Reply:
Thanks! I put a ? there to remember to verify the text xD